mangaki/mangaki/utils/anidb.py

```diff
@@ -500,8 +500,8 @@ def get_or_update_work(self,
         return work
 
 client = AniDB(
-    getattr(settings, 'ANIDB_CLIENT', None),
-    getattr(settings, 'ANIDB_VERSION', None)
+    getattr(settings, 'ANIDB_CLIENT', 'test_client'),
```

Comment:
Why do you need to do this?
If, for test purposes, use appropriate decorators to modify the settings or patch the client using mocks.

Reply:
Yes, it was because of client. Such a thing should not exist. I added a FIXME.